tests/test_parquet.py

```diff
@@ -634,6 +634,29 @@ def testAstropyTable(self):
         with self.assertRaises(ValueError):
             self.butler.get(self.datasetType, dataId={}, parameters={"columns": ["e"]})
 
+    def testAstropyTableWithMetadata(self):
+        tab1 = _makeSimpleAstropyTable(include_multidim=True)
+
+        meta = {
+            "meta_a": 5,
+            "meta_b": 10.0,
+            "meta_c": [1, 2, 3],
+            "meta_d": True,
+            "meta_e": "string",
+        }
+
+        tab1.meta.update(meta)
+
+        self.butler.put(tab1, self.datasetType, dataId={})
+        # Read the whole Table.
+        tab2 = self.butler.get(self.datasetType, dataId={})
+        self._checkAstropyTableEquality(tab1, tab2)
+
+        # Check the metadata
```

Comment:
Should this maybe go in `_checkAstropyTableEquality` (perhaps with a `checkMeta` kwarg)? And is there a problem with `tab1.meta == tab2.meta`?